webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/Virtualization/VirtEc2.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+import { translate as __ } from '../../../../../../common/I18n';
+
+const VirtEc2 = ({ vm }) => (
+  <>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Availability zone')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {vm.availability_zone}
+      </DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Public ip address')}</DescriptionListTerm>
```

Comment:
I'd capitalize `IP` since it's an abbreviation
```suggestion
      <DescriptionListTerm>{__('Public IP address')}</DescriptionListTerm>
```